lib/db_utils.go

```diff
@@ -476,8 +477,50 @@ type DBPrefixes struct {
 	// 	<prefix, expirationBlockHeight, PKID, partialID> -> <>
 	PrefixNoncePKIDIndex []byte `prefix_id:"[77]" is_state:"true"`
 
-	// NEXT_TAG: 78
+	// PrefixTxnHashToTxn is used to store transactions that have been processed by the node. This isn't actually stored
+	// in badger, but is tracked by state syncer when processing mempool transactions.
+	// The index has the following structure:
+	// 	<prefix, txnHash> -> <Transaction>
+	PrefixTxnHashToTxn []byte `prefix_id:"[78]" core_state:"true"`
+
+	// PrefixTxnHashToUtxoOps is used to store UtxoOps for transactions that have been processed by the node.
+	// This isn't actually stored in badger, but is tracked by state syncer when processing mempool transactions.
+	PrefixTxnHashToUtxoOps []byte `prefix_id:"[79]" core_state:"true"`
```

Comment:
It looks like this is only used in the unconnectedTransactions thing in the mempool dumping part, which I think we can delete, which would then mean that we can delete this as well.